guides/common/assembly_administering-project-with-hammer.adoc

```diff
@@ -0,0 +1,15 @@
+[id="administering-project-with-hammer_{context}"]
+= Getting started with using Hammer to manage {Project}
+
+Get to know the basics of administering {Project} from the command line by using the Hammer tool.
+
+include::modules/ref_getting-help-with-hammer-in-project.adoc[leveloffset=+1]
+
+include::modules/con_hammer-cli-overview.adoc[leveloffset=+1]
+
+== Additional resources
+
```

Comment:
```suggestion
.Additional resources
```

alternative: move to separate "snip_" file. But by convension, we should not have any heading/text in assemblies.